pymongoexplain/commands.py

```diff
@@ -19,7 +19,6 @@
 from typing import Union
 
 from bson.son import SON
-from bson.py3compat import abc, iteritems, string_type
```

Comment:
You should replace `from bson.py3compat import abc` with `from collections import abc` instead of using dict/list/set directly.